raster-foundry/raster-foundry
# Add a migration to add annotation project id to tasks

## Overview

This PR adds a migration to add a `annotation_project_id` field to `tasks` table and populates this field with `id`s of `annotation_projects` project. 

### Checklist

- ~Swagger specification updated~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

### Demo

```
$ ./scripts/migrate migrate
[RF-API] Running migrations with docker-compose
Starting raster-foundry_postgres_1 ... done
Flyway Community Edition 5.2.4 by Boxfuse
Database: jdbc:postgresql://database.service.rasterfoundry.internal/ (PostgreSQL 9.6)
Successfully validated 33 migrations (execution time 00:00.450s)
Current version of schema "public": 23
Migrating schema "public" to version 24 - update task annotation project id
Successfully applied 1 migration to schema "public" (execution time 00:00.193s)
```

## Testing Instructions

- Follow steps here: https://github.com/raster-foundry/raster-foundry/pull/5286 if there is no annotation project in your DB
- `./scripts/migrate migrate`
- make sure it works and make sure it adds a new `annotation_project_id` column to `tasks` column with right constraints and has values populated

Closes https://github.com/raster-foundry/raster-foundry/issues/5289
